.golangci.yml

```diff
@@ -0,0 +1,26 @@
+# WARNING: This file is autogenerated - changes will be overwritten if not made via https://github.com/pulumi/ci-mgmt
```

Comment:
Is this copied from somewhere. I was pretty sure we didn't have ci-mgmt enabled for non-provider repos.

Reply:
Good catch; yes this started out as a copy from a provider repo